pkg/gds/service_test.go

```diff
@@ -203,16 +204,25 @@ func (s *gdsTestSuite) TestFixtures() {
 	expected := map[fixtureType]map[string]int{
 		empty: {},
 		small: {vasps: 2},
-		full:  {vasps: 14, certreqs: 10},
+		full:  {vasps: 14, certs: 3, certreqs: 10},
 	}
 
 	// Test the reference fixtures
-	require.Len(s.fixtures, 2)
+	require.Len(s.fixtures, 3)
 	require.Contains(s.fixtures, vasps)
+	require.Contains(s.fixtures, certs)
 	require.Contains(s.fixtures, certreqs)
 	require.Len(s.fixtures[vasps], expected[full][vasps])
+	require.Len(s.fixtures[certs], expected[full][certs])
 	require.Len(s.fixtures[certreqs], expected[full][certreqs])
 
+	// Validate VASP fixtures
+	for name, obj := range s.fixtures[vasps] {
+		vasp, ok := obj.(*pb.VASP)
+		require.True(ok, "could not marshal VASP record %s", name)
+		require.NoError(vasp.Validate(true), "VASP record %s is invalid", name)
+	}
```

Comment:
`DeleteContact` does VASP validation so I added a check here to make sure that all our fixtures are valid.

Reply:
Thank you!